log-window.js

```diff
@@ -158,6 +170,23 @@ class EmeLoggerWindow {
     return '0x' + byte.toString(16).padStart(2, '0');
   }
 
+  /**
+   * @param {number} byte
+   * @return {string}
+   * @private
+   */
+  bytesToBase64_(bytes) {
```

Comment:
I don't see any unit test for the existing `byteToHex_` and for this new `bytesToBase64_` in /spec/log-window-tests.js. Can we add new unit tests for them?

Reply:
done.